### PR TITLE
Documentation changes in preparation for Mattermost Marketplace submittal

### PR DESCRIPTION
**Pull request by mkdbns**
…ttal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,109 @@
-# mattermost-plugin-exchange-ews-proxy
+# Mattermost Microsoft Exchange Calendar EWS Proxy
 
 ## Table of Contents
 
-1. [Overview](#overview)
-2. [Installation and Setup](#installation-and-setup)
-3. [Getting started](#getting-started)
+- [License]($license)
```
Comment (shivamjosh) [inline README.md:5]:
```suggestion
- [License](#license)
```

File: README.md [hunk 2]
```diff
@@ -1,62 +1,109 @@
...
+
+## License
+
+See the [LICENSE](./LICENSE) file for license rights and limitations.
 
 ## Overview
-An intermediate server made in Java to add capabilities for two-way interaction between Mattermost plugin and Microsoft Outlook Calendar. This server acts as a proxy and connects Mattermost Exchange MS-Calendar Plugin with Microsoft Outlook Calendar.
 
-## Installation and Setup
+The Microsoft Exchange Calendar EWS Proxy is a JAVA-based intermediary service which enables the  [Mattermost Microsoft Exchange Calendar Plugin](https://github.com/Brightscout/mattermost-plugin-exchange-calendar) to communicate with on-premise Microsoft Exchange servers.
```
Comment (shivamjosh) [inline README.md:16]:
```suggestion
The Microsoft Exchange Calendar EWS Proxy is a JAVA-based intermediary service that enables the [Mattermost Microsoft Exchange Calendar Plugin](https://github.com/Brightscout/mattermost-plugin-exchange-calendar) to communicate with on-premise Microsoft Exchange servers.
```

File: README.md [hunk 3]
```diff
@@ -1,62 +1,109 @@
...
+### Build and Run the JAR
 
-## Getting started
+Run the following commands to prepare a distributable JAR:
 
-#### Try it out with Java
+```bash
+git clone git@github.com:Brightscout/mattermost-plugin-exchange-ews-proxy.git
+cd mattermost-plugin-exchange-ews-proxy
+./gradlew build
```
Comment (shivamjosh) [inline README.md:93]: When we build using `java` we need the environment variables exported to the shell. So should we mention that step here?

**Review by shivamjosh — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,62 +1,109 @@
...
+| EWS_USERNAME | Username of the Exchange service account.|
+| EWS_PASSWORD | Password of the Exchange service account.|
+| EWS_SECRET_AUTH_KEY | Secret used to authenticate API requests to this server.|
+| EWS_SUBSCRIPTION_LIFETIME_IN_MINUTES | Length of time that Exchange will tolerate delivery failures before cancelling subscriptions for a user.|
+| JAR_PATH | Path of the JAR file which is the server's executable.|
+| SERVER_PORT | Port on which the server listen on.|
+
+### Run with Docker
+
+The simplest way to run this service is to do so via Docker. A public [Docker image is available on Docker Hub](https://hub.docker.com/layers/mattermost-plugin-exchange-ews-proxy/brightscout/mattermost-plugin-exchange-ews-proxy/latest/images/sha256-f1172b134258a2bf9d75231ba4c7fd92c8161f7a4ffe4ec16adb3ea74b977e5c?context=explore).
```
Comment (manojmalik20) [inline README.md:48]:
```suggestion
The simplest way to run this service is to do so via Docker. A public [Docker image](https://hub.docker.com/layers/mattermost-plugin-exchange-ews-proxy/brightscout/mattermost-plugin-exchange-ews-proxy/latest/images/sha256-f1172b134258a2bf9d75231ba4c7fd92c8161f7a4ffe4ec16adb3ea74b977e5c?context=explore) is available on Docker Hub.
```

File: README.md [hunk 5]
```diff
@@ -1,62 +1,109 @@
...
-* **SERVER_PORT** : Port on which the server will run.
-* **SUBSCRIPTION_LIFETIME_IN_MINUTES** : Expiry time of the subscription to push notifications in minutes.
+    ```bash
+     java -jar mattermost-plugin-exchange-ews-proxy-<version>.jar
+     ```
+
+## Development
+
+Make sure you have the following components installed:
+  - [JDK 6+](https://openjdk.java.net/install/) or later
```
Comment (manojmalik20) [inline README.md:81]:
```suggestion
  - [JDK 6](https://openjdk.java.net/install/) or later
```

**Review by manojmalik20 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (mkdbns) [reply README.md:96]: Added text for this.

**Pull request merged**